2020-11-17 22:38:04,542	INFO services.py:1092 -- View the Ray dashboard at [1m[32mhttp://127.0.0.1:8265[39m[22m
2020-11-17 22:38:19,946	WARNING worker.py:1091 -- The dashboard on node cx3-2-5.cx3.hpc.ic.ac.uk failed with the following error:
Traceback (most recent call last):
  File "/rds/general/user/asm119/home/anaconda3/envs/reinforcement/lib/python3.7/site-packages/ray/dashboard/dashboard.py", line 962, in <module>
    dashboard.run()
  File "/rds/general/user/asm119/home/anaconda3/envs/reinforcement/lib/python3.7/site-packages/ray/dashboard/dashboard.py", line 577, in run
    aiohttp.web.run_app(self.app, host=self.host, port=self.port)
  File "/rds/general/user/asm119/home/anaconda3/envs/reinforcement/lib/python3.7/site-packages/aiohttp/web.py", line 508, in run_app
    loop.run_until_complete(main_task)
  File "/rds/general/user/asm119/home/anaconda3/envs/reinforcement/lib/python3.7/asyncio/base_events.py", line 579, in run_until_complete
    return future.result()
  File "/rds/general/user/asm119/home/anaconda3/envs/reinforcement/lib/python3.7/site-packages/aiohttp/web.py", line 411, in _run_app
    await site.start()
  File "/rds/general/user/asm119/home/anaconda3/envs/reinforcement/lib/python3.7/site-packages/aiohttp/web_runner.py", line 128, in start
    reuse_port=self._reuse_port,
  File "/rds/general/user/asm119/home/anaconda3/envs/reinforcement/lib/python3.7/asyncio/base_events.py", line 1374, in create_server
    % (sa, err.strerror.lower())) from None
OSError: [Errno 98] error while attempting to bind on address ('127.0.0.1', 8265): address already in use

WARNING:tensorflow:From /rds/general/user/asm119/home/anaconda3/envs/reinforcement/lib/python3.7/site-packages/tensorflow/python/compat/v2_compat.py:96: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
2020-11-17 22:38:56,038	INFO trainer.py:592 -- Tip: set framework=tfe or the --eager flag to enable TensorFlow eager execution
2020-11-17 22:38:56,038	INFO trainer.py:1065 -- `_use_trajectory_view_api` only supported for PyTorch so far! Will run w/o.
2020-11-17 22:38:56,038	INFO trainer.py:619 -- Current log_level is WARN. For more information, set 'log_level': 'INFO' / 'DEBUG' or use the -v and -vv flags.
2020-11-17 22:39:28,048	INFO trainable.py:255 -- Trainable.setup took 32.011 seconds. If your trainable is slow to initialize, consider setting reuse_actors=True to reduce actor creation overheads.
2020-11-17 22:39:28,048	WARNING util.py:40 -- Install gputil for GPU system monitoring.
WARNING:tensorflow:From /rds/general/user/asm119/home/anaconda3/envs/reinforcement/lib/python3.7/site-packages/ray/rllib/policy/tf_policy.py:877: Variable.load (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Prefer Variable.assign which has equivalent behavior in 2.X.
